docs/en/status-communities/about-voting-to-change-community-visibility.md

```diff
@@ -0,0 +1,67 @@
+---
+id: 685
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# About voting to change community visibility
+
+All SNT holders can engage in [voting to include or exclude a community][vote-to make-a-community-private-or-public] from the Communities Discover page, and [voting to feature a community][vote-to-feature-a-community]. 
+
+Votes happen on the [community directory curation dApp][curate.status.app]. This is an independent, decentralized application deployed on the Optimism network and is separate from the Status app. Voting outcomes are solely determined by the voters, and Status has no control over these results.
+
+When a community is created, it's private by default, which is ideal for small, exclusive communities. However, if the community targets a broad audience, you might prefer to make it public.
```

Comment:
Technically, the community is never private. An unencrypted community description is broadcasted on the Waku network as soon as it's created, implying that anyone can read it.

Perhaps instead of saying "it's private by default," we could state that "it is not listed in the global directory."

Reply:
We name "communities that are not listed in the global directory" as "private communities". I'll rephrase this sentence to avoid confusion.